autoPyTorch/utils/backend.py

```diff
@@ -328,6 +328,11 @@ def load_datamanager(self) -> BaseDataset:
             with open(filepath, 'rb') as fh:
                 return pickle.load(fh)
 
+    def replace_datamanager(self, datamanager: BaseDataset) -> None:
```

Comment:
@ravinkohli There probably should be a condition to call replace manager only when search is done.

Reply:
yeah I'm now handling this better.